src/RealtimeEffectPanel.cpp

```diff
@@ -290,8 +288,8 @@ namespace
       std::shared_ptr<RealtimeEffectState> mEffectState;
       std::shared_ptr<EffectSettingsAccess> mSettingsAccess;
 
-      ThemedButtonWrapper<wxButton>* mChangeButton{nullptr};
-      ThemedButtonWrapper<wxBitmapButton>* mEnableButton{nullptr};
+      ThemedAButtonWrapper<AButton>* mChangeButton{nullptr};
```

Comment:
The commit "RealtimeEffectPanel wx*Button replaced with AButton" doesn't build.  The next commit repairs that.  Make both build.